src/components/ApplicationGroup/AppGroupAppFilter.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from 'react'
+import ReactSelect, { components, MultiValue } from 'react-select'
+import { ReactComponent as ShowIcon } from '../../assets/icons/ic-visibility-on.svg'
```

Comment:
assets import must be at the bottom(as per FE guideline)

Reply:
Done